oci/oci.go

```diff
@@ -0,0 +1,60 @@
+package oci
+
+import (
+	"path/filepath"
+	"strings"
+
+	"github.com/mrunalp/ocid/utils"
+)
+
+// New creates a new Runtime with options provided
+func New(name string, runtimePath string, sandboxDir string) (*Runtime, error) {
```

Comment:
If we just pass `runc` in https://github.com/mrunalp/ocid/pull/19/files#r72885893 this func doesn't need to receive 2 identical args (`runtimePath` and `name` should be the same, is there any runtime which has this differ?